src/test/java/com/carhouse/config/TestConfig.java

```diff
@@ -0,0 +1,34 @@
+package com.carhouse.config;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.github.tomakehurst.wiremock.WireMockServer;
+import org.springframework.context.annotation.*;
+import org.springframework.web.client.RestTemplate;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.configureFor;
+import static com.github.tomakehurst.wiremock.core.WireMockConfiguration.options;
+
+@Configuration
+@ComponentScan("com.carhouse.provider")
+@PropertySources({@PropertySource("classpath:endpoints.properties"),
+@PropertySource("classpath:test.properties")})
+public class TestConfig {
+
+    @Bean
+    public RestTemplate getRestTemplate() {
+        return new RestTemplate();
+    }
+
+    @Bean
+    public ObjectMapper getObjectMapper() {
+        return new ObjectMapper();
+    }
+
+    @Bean
+    public void initializeWireMockServer() {
+        WireMockServer wireMockServer = new WireMockServer(options()
+                .bindAddress("localhost").port(65199));
```

Comment:
Are you really feel comfortable that you HAD TO change `WireMockServer` port in 2 places?
Are you satisfied with that?
Are you really comfortable to look through tons of configs while you're trying to find place there this shitty configs has been missed?

Reply:
I'll add propery